TicketBooking/Flight/FlightRepository/FlightRepository.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using TicketBooking.Flight.FlightModel;
+using TicketBooking.Country;
+using TicketBooking.Class;
+using TicketBooking.Airport;
+using TicketBooking.Flight.Services;
+
+namespace TicketBooking.Flight.FlightRepository
+{
+    public class FlightRepository
+    {
+        public List<FlightModel.Flight> Flights { get; private set; } = new List<FlightModel.Flight>();
```

Comment:
`private readonly _flightsList`.

In repositoties, we expose only functions, we usually don't expose variables.